addons/tags/tags.php

```diff
@@ -459,7 +459,7 @@ public function page_title( $title ) {
 	public function ap_breadcrumbs( $navs ) {
 		if ( is_question_tag() ) {
 			$tag_id       = sanitize_title( get_query_var( 'q_tag' ) );
-			$tag = get_term_by( 'slug', $tag_id, 'question_tag' ); // @codingStandardsIgnoreLine.
+			$tag          = $tag_id ? get_term_by( 'slug', $tag_id, 'question_tag' ) : get_queried_object();
```

Comment:
@mebishalnapit Add if check after this to make sure tag is an object or term.

Reply:
@rahularyan It should already be available here: https://github.com/anspress/anspress/blob/bc169e9242ce91dfeceb0642db304408352a2e88/addons/tags/tags.php#L469-L475